### PR TITLE
Master pivot fix pro

### PR DESCRIPTION
**Pull request by pro-odoo**
There are two issues since d06e67e748ca9c5c538d5e1a1efc4ae798fee707:
- Owl renderer was not destroyed
- on_attach_callback and on_detach_callback were not called on super, so not applied to the controlPanel and searchPanel.

### CODE REVIEW

File: addons/web/static/tests/views/abstract_controller_tests.js [hunk 1]
```diff
@@ -123,5 +123,23 @@ QUnit.module("Views", {
         assert.verifySteps(["destroy"]);
 
     });
+
+    QUnit.test('Correctly set focus to search panel with Owl Renderer', async function (assert) {
+        assert.expect(1);
+
+        class Renderer extends AbstractRenderer { }
+        Renderer.env = makeTestEnvironment({});
```
Comment (aab-odoo) [inline addons/web/static/tests/views/abstract_controller_tests.js]: not necessary i think (in both tests)

Comment (aab-odoo) [reply addons/web/static/tests/views/abstract_controller_tests.js]: don't forget to remove import if so

Comment (pro-odoo) [reply addons/web/static/tests/views/abstract_controller_tests.js]: done :)

File: addons/web/static/tests/views/abstract_controller_tests.js [hunk 2]
```diff
@@ -78,5 +98,45 @@ QUnit.module("Views", {
...
+
+        class Renderer extends AbstractRenderer { }
+        Renderer.template = xml`<div>Test</div>`;
+
+        var view = await createView({
+            View: getOwlView(Renderer, "test"),
+            data: this.data,
+            model: "test_model",
+            arch: "<test/>",
+            debug: 1,
```
Comment (aab-odoo) [inline addons/web/static/tests/views/abstract_controller_tests.js]: :D

**Comment by aab-odoo** [comment]
robodoo rebase-ff

**Comment by aab-odoo** [comment]
robodoo r+

**Pull request closed without merging**